### PR TITLE
CRAW-216: Change the way of using Jsoup CSS Selectors

### PR DESCRIPTION
**Pull request by abhishekvm**
1. Replaced element[class=someclass] to element.classname
2. Replaced *[class=someclass] and [class=someclass] to .classname
3. Replaced \*[class\*=someclass] to [class*=someclass]
   - The starting * in \*[class\*=someclass] is redundant and does not create any differnce
4. Replaced *[id=someid] to [id=someid]
5. Replaced \*[id\*=someid] to [id*=someid]

We were expected to have some performance improvement as mentioned on https://airpr1.atlassian.net/browse/CRAW-216. After the change, I cross checked the performance and don't see any difference. Extractor takes approximately the same time before and after the change.

### CODE REVIEW

File: src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java [hunk 1]
```diff
@@ -133,66 +133,66 @@ public boolean hasHTMLTags(String text){
...
         aMap.put("theverge.com", Arrays.asList(
-                "*[class=m-linkset__entries-item]",
-                "*[class=m-linkset]",
-                "*[class=\"feature-photos-story feature-photos-column\"]",
-                "*[class*=js-carousel-pane]",
-                "*[id=feature-photos-model]"
+                ".m-linkset__entries-item",
+                ".m-linkset",
+                ".feature-photos-story.feature-photos-column",
+                "[class*=js-carousel-pane]",
```
Comment (skyshard) [inline src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java]: are we intentionally matching on partial class names or would this just be equivalent to `.js-carousel-pane`?

Comment (abhishekvm) [reply src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java]: Need to check this out may be by changing to .classname and running testcases.

**Review by andresp99999 — APPROVED** [review]
LGTM, even if performance difference is small, I think rules are easier to read this way.

**Comment by andresp99999** [comment]
@skyshard, are you ok with merging this branch?

**Comment by skyshard** [comment]
LGTM overall. There seems to be many cases of `[class*=something]` still where I'm not sure that we actually intend that behavior (wildcard matching anywhere within the class attribute, vs just matching any of the individual classes). We can address that in a different pull

**Pull request merged**